src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -51,7 +51,7 @@ export default function ConfirmedAppointmentListItem({
   const isInPersonVAAppointment = !isVideo && !isCommunityCare;
 
   const showInstructions =
-    isCommunityCare ||
+    (isCommunityCare && appointment.contained[0].address) ||
```

Comment:
This doesn't get the correct result on the page, but it feels like a rule that will be confusing later, since it reads like "we don't show instructions for community care appointments that don't have an address." Which is incidentally true, but not really the root of what we're trying to do.

I wonder if it would be better to exclude these comments in the transformer? We could have some logic in here to only use `instructionsToVeteran` for `comment` for CC appts.

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/services/appointment/transformers.js#L525

Reply:
Good idea about using the comment field in the transformer. I placed a ternary statement to check for `communityCare` so the alert text is passed to the `communityCareInstructions` component